librz/debug/p/debug_native.c

```diff
@@ -1450,6 +1450,9 @@ static int rz_debug_native_bp(RzBreakpoint *bp, RzBreakpointItem *b, bool set) {
 		return set
 			? arm32_hwbp_add(dbg, bp, b)
 			: arm32_hwbp_del(dbg, bp, b);
+#elif __riscv
```

Comment:
Added the elif so that we would not get a compiler warning for unused dbg for riscv